north/src/runtime/state.rs

```diff
@@ -315,36 +314,42 @@ impl State {
         }
     }
 
-    pub async fn shutdown(&self) -> result::Result<(), Error> {
-        if self
+    /// Send a shutdown request to the main loop
+    pub async fn initiate_shutdown(&mut self) {
+        self.events_tx
+            .send(Event::Shutdown)
+            .await
+            .expect("Internal channel error on main");
+    }
+
+    /// Shutdown the runtime: stop running applications and umount npks
+    pub async fn shutdown(&mut self) -> result::Result<(), Error> {
```

Comment:
Do we still need this in the public interface since we have `initiate_shutdown`?

Reply:
Think so because it's used from the console.